### ISSUE TITLE
Riak CS fails to start on Ubuntu 11.10 after installing with .deb

### ISSUE THREAD

**Issue by dreverri** [basho/riak_cs#171]
## Steps to reproduce
- Install `libssl0.9.8`

``` bash
$ sudo apt-get install libssl0.9.8
```
- Download the `riak-cs_1.0.0-1_amd64.deb` package
- Install the package

``` bash
$ sudo dpkg -i riak-cs_1.0.0-1_amd64.deb
```
- Attempt to start Riak CS

``` bash
$ sudo riak-cs start
/usr/sbin/riak-cs: line 8: /usr/lib/riak-cs/lib/env.sh: Permission denied
/usr/sbin/riak-cs: line 11: check_user: command not found
/usr/sbin/riak-cs: line 19: node_down_check: command not found
mkdir: missing operand
Try `mkdir --help' for more information.
/usr/sbin/riak-cs: line 25: /run_erl: No such file or directory
```
- Check permissions on `/usr/lib/riak-cs/lib/env.sh`

``` bash
$ ls -al /usr/lib/riak-cs/lib/env.sh
-rw------- 1 root root 2302 2012-04-02 13:23 /usr/lib/riak-cs/lib/env.sh
```
- Update permissions to allow read access to all users

``` bash
$ sudo chmod -R a+r /usr/lib/riak-cs/lib/env.sh
```
- Attempt to start Riak CS

``` bash
$ sudo riak-cs start
grep: /etc/riak-cs/vm.args: Permission denied
vm.args needs to have either -name or -sname parameter.
```
- Check permissions on `/etc/riak-cs/`

``` bash
$ ls -al /etc/riak-cs/
total 28
drwxr-xr-x  2 root root 4096 2012-04-14 19:51 .
drwxr-xr-x 82 root root 4096 2012-04-14 19:51 ..
-rw-------  1 root root 4512 2012-04-02 13:23 app.config
-rw-------  1 root root 1009 2012-04-02 13:23 cert.pem
-rw-------  1 root root  887 2012-04-02 13:23 key.pem
-rw-------  1 root root 1125 2012-04-02 13:23 vm.args
```
- Update permissions to allow read access to all users

``` bash
$ sudo chmod -R a+r /etc/riak-cs/
```
- Attempt to start Riak CS

``` bash
$ sudo riak-cs start
escript: Failed to open file: /usr/lib/riak-cs/erts-5.8.5/bin/nodetool
```
- Check permissions on `/usr/lib/riak-cs/erts-5.8.5/bin`

``` bash
$ ls -al /usr/lib/riak-cs/erts-5.8.5/bin
total 4532
drwxr-xr-x 2 root root    4096 2012-04-14 19:51 .
drwxr-xr-x 8 root root    4096 2012-04-14 19:51 ..
-rwxr-xr-x 1 root root 2053264 2012-04-02 13:24 beam
-rwxr-xr-x 1 root root 2249808 2012-04-02 13:24 beam.smp
-rwxr-xr-x 1 root root   14504 2012-04-02 13:24 child_setup
-rwxr-xr-x 1 root root   26928 2012-04-02 13:24 ct_run
-rwxr-xr-x 1 root root   47632 2012-04-02 13:24 epmd
-rwxr-xr-x 1 root root    1182 2012-04-02 13:23 erl
-rwxr-xr-x 1 root root   31056 2012-04-02 13:24 erlc
-rwxr-xr-x 1 root root   48048 2012-04-02 13:24 erlexec
-rwxr-xr-x 1 root root   26992 2012-04-02 13:24 escript
-rwxr-xr-x 1 root root   10360 2012-04-02 13:24 heart
-rwxr-xr-x 1 root root   47568 2012-04-02 13:24 inet_gethost
-rw------- 1 root root    4874 2012-04-02 13:23 nodetool
-rwxr-xr-x 1 root root   22960 2012-04-02 13:24 run_erl
-rwxr-xr-x 1 root root    1183 2012-04-02 13:23 start
-rwxr-xr-x 1 root root   14488 2012-04-02 13:24 to_erl
```
- Update permissions to allow read and execute access to all users

``` bash
sudo chmod a+rx /usr/lib/riak-cs/erts-5.8.5/bin/nodetool
```
- Attempt to start Riak CS

``` bash
$ sudo riak-cs start
```
## Expected

Riak CS would start without issue the first time
## Actual

Several permission settings had to be adjusted before Riak CS could be started

**Comment by jaredmorrow** [comment]
Fixed in node_package with release 0.2.1 https://github.com/basho/node_package/pull/8

Will test on next again on next release of riak-cs before closing this issue.

**Comment by jaredmorrow** [comment]
Confirmed fixed Ubuntu 10.04 with Lucid package and riak-cs 1.0.1 build

Additional postinst script work was needed on the debian package in node_package: https://github.com/basho/node_package/commit/e8af2674a4824178e354b92b9bd2a1d9a2e07d91
